### ISSUE TITLE
Avatar in admin bar breaks out

### ISSUE THREAD

**Issue by franz-josef-kaiser** [wecodemore/wcm-avatar#15]
The currently logged in user avatar currently breaks out of the admin bar if s/he has a custom avatar. The desired Markup as used with the _Gravatar_ service is:

``` html
<a class="ab-item" aria-haspopup="true" href="https://example.com/wp/wp-admin/profile.php">
    Howdy, admin
    <img alt="" src="https://1.gravatar.com/avatar/d402074cd4c1b928f1a3489d83b8ce15?s=26&amp;d=mm&amp;r=g" srcset="https://1.gravatar.com/avatar/d402074cd4c1b928f1a3489d83b8ce15?s=52&amp;d=mm&amp;r=g 2x" class="avatar avatar-26 photo" height="26" width="26">
</a>
```

… while the user has the following Markup with a custom avatar:

``` html
<a class="ab-item" aria-haspopup="true" href="https://nzz.wp/wp/wp-admin/profile.php">
    Howdy, admin
</a>
<a href="https://nzz.wp/wp/wp-admin/post.php?post=1988&amp;action=edit">
    <img width="26" height="26" src="https://nzz.wp/wp-content/uploads/2015/08/avatar.jpeg" class="" alt="avatar" user_avatar="1">
</a>
```

**Comment by franz-josef-kaiser** [comment]
Admin Bar icon avatar fixed with ae5d5a1

Still the hover menu has the full size.

**Comment by franz-josef-kaiser** [comment]
Done
